docs/docsite/rst/getting_started_ee/introduction.rst

```diff
@@ -55,8 +55,8 @@ Projects in the Ansible ecosystem also provide lots of tooling that you can use
 
 * `Ansible Builder <https://ansible-builder.readthedocs.io/en/stable/>`_
 * `Ansible Navigator <https://ansible-navigator.readthedocs.io/>`_
-* `Ansible AWX <https://docs.ansible.com/automation-controller/latest/html/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_
-* `Automation controller <https://docs.ansible.com/automation-controller/latest/html/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_
+* `Ansible AWX <https://docs.ansible.com/automation-control node/latest/html/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_
```

Comment:
```suggestion
* `Ansible AWX <https://ansible.readthedocs.io/projects/awx/en/latest/userguide/execution_environments.html#use-an-execution-environment-in-jobs>`_
```

Reply:
I'm just taking the opportunity to fix a URL in this one :-)  We just published new AWX docs.